### PR TITLE
Tomb of Sargeras skeleton

### PR DESCRIPTION
**Pull request by Justw8**

### CODE REVIEW

**Comment by funkydude** [comment]
Please rename `of` to a capital O.
You could also add the entries into pkgmeta and modules.xml in the root directory.

File: TombofSargeras/BigWigs_TombofSargeras.toc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Interface: 70200
+
+## Title: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-deDE: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-esES: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-esMX: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-frFR: BigWigs [|cffeda55fTomb of Sargerast|r]
```
Comment (funkydude) [inline TombofSargeras/BigWigs_TombofSargeras.toc]: Typo

File: TombofSargeras/BigWigs_TombofSargeras.toc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Title-zhTW: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-itIT: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-ptBR: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Title-koKR: BigWigs [|cffeda55fTomb of Sargeras|r]
+## Notes: BigWigs Tomb of Sargeras Modules
+## Notes-deDE: BigWigs-Module für Tomb of Sargeras
+## Notes-esES: BigWigs Tomb of Sargeras Modules
+## Notes-esMX: BigWigs Tomb of Sargeras Modules
+## Notes-frFR: Modules BigWigs concernant Tomb of Sargeras.
+## Notes-ruRU: Модуль BigWigs для Цитадели Ночи
```
Comment (funkydude) [inline TombofSargeras/BigWigs_TombofSargeras.toc]: Copy/paste fragment

Comment (Justw8) [inline TombofSargeras/BigWigs_TombofSargeras.toc]: I don't see it in the lines you quoted. Did notice some below that.

Comment (funkydude) [inline TombofSargeras/BigWigs_TombofSargeras.toc]: > I don't see it in the lines you quoted. Did notice some below that.

Notes-ruRU is a copy/paste from Nighthold.
Also you shouldn't be using a capital o in the toc field, that's the ingame proper-English display, thanks.

**Pull request merged**